web/client/components/data/download/DownloadOptions.jsx

```diff
@@ -60,6 +74,12 @@ class DownloadOptions extends React.Component {
 
     render() {
         return (<form>
+            <label><Message msgId="layerdownload.service" /></label>
+            <Select
+                value={this.props.service}
+                disabled={!this.props.wpsAvailable}
+                onChange={(sel) => this.props.setService(sel.value)}
+                options={this.services} />
```

Comment:
I suggest to hide the select input when wps is not available Instead of  disabled it